src/components/Footer/ListLinks.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Link from './Link';
+
+const ListLinks = ({classNameContainer, classNameChild, children, links}) => (
```

Comment:
Let's change files with jsx to be `script.jsx` and not `script.js`

Reply:
it's not obligatory